modules/peers.js

```diff
@@ -520,16 +589,28 @@ Peers.prototype.onPeersReady = function () {
 			},
 			updatePeers: function (seriesCb) {
 				var updated = 0;
-				var peers = library.logic.peers.list();
+				var peers = self.acceptable(library.logic.peers.list());
```

Comment:
Why acceptable? We shouldn't have any peers in list that doesn't already satisfied acceptable.